hooks/repo.go

```diff
@@ -50,7 +51,7 @@ type hookRepository struct {
 	auctionResponseHooks         map[string]hookstage.AuctionResponse
 }
 
-func (r *hookRepository) GetEntrypointHook(id string) (h hookstage.Entrypoint, ok bool) {
+func (r *hookRepository) GetEntrypointHook(id string) (hookstage.Entrypoint, bool) {
```

Comment:
Named parameters not used in this function and other funtions in this file don't come with returned named parameters at all.